components/anchor/anchor-link.component.ts

```diff
@@ -74,12 +81,11 @@ export class NzAnchorLinkComponent implements OnInit, OnDestroy {
   @ContentChild('nzTemplate', { static: false }) nzTemplate!: TemplateRef<void>;
   @ViewChild('linkTitle') linkTitle!: ElementRef<HTMLAnchorElement>;
 
-  constructor(
-    public elementRef: ElementRef,
-    private anchorComp: NzAnchorComponent,
-    private platform: Platform,
-    private renderer: Renderer2
-  ) {}
+  constructor() {
+    this.destroyRef.onDestroy(() => {
```

Comment:
There are cases like this one, where `destroyRef` is used exclusively in the constructor, where I feel a constructor parameter is always preferred, instead of keeping a reference on each component instance. Or a local `inject` call.